modules/462-loki/openapi/doc-ru-config-values.yaml

```
@@ -40,49 +36,75 @@ properties:
       Структура, аналогичная `spec.tolerations` пода Kubernetes.
 
       Если ничего не указано или указано `false`, будет [использоваться автоматика](https://deckhouse.ru/documentation/v1/#выделение-узлов-под-определенный-вид-нагрузки).
-  resourcesRequests:
+  resourcesManagement:
     description: |
-      Настройки максимальных значений CPU и памяти, которые может запросить под при выборе узла.
-
-      Если модуль [vertical-pod-autoscaler](../302-vertical-pod-autoscaler/) выключен, максимальные значения становятся значениями по умолчанию.
+      Варианты управления ресурсами Loki.
     properties:
       mode:
-        description: |
-          Режим управления реквестами ресурсов.
+        description: >
+          Режим управления запросами ресурсов. Классические `Static`
+          запросы/ограничение или `VPA`.
       vpa:
         description: |
-          Настройки режима управления VPA.
+          Параметры VPA.
         properties:
           mode:
             description: |
-              Режим работы VPA.
+              Режим VPA.
           cpu:
             description: |
-              Настройки для CPU.
+              Параметры, связанные с процессором.
             properties:
               max:
                 description: |
-                  Максимальное значение, которое может выставить VPA для реквеста к CPU.
+                  Максимально допустимые запросы к процессору.
               min:
                 description: |
-                  Минимальное значение, которое может выставить VPA для реквеста к CPU.
+                  Минимально допустимые запросы к процессору.
+              limitRatio:
+                description: >
+                  Коэффициент расчета лимитов процессора.
+                  Необязательно используется для расчета начальных лимитов для пода.
+                  VPA, со своей стороны, сохраняет соотношение начальных лимитов/запросов
+                  при дальнейшей настройке ресурсов.
           memory:
             description: |
-              Значение для реквеста к памяти.
+              Запрашиваемый объем памяти.
```

```suggestion
              Настройки VPA при работе с памятью.
```

Это если править согласно комментария про CPU

---

Ок. Раз там поменялось, тут тоже)